data/campaigns/Eastern_Invasion/utils/_final.cfg

```diff
@@ -0,0 +1,906 @@
+#textdomain wesnoth-ei
```

Comment:
I think i'm missing something really obvious here but how does this work? Where are these events actually included?

Reply:
Included in the scenario files for S17a and S17b.

Reply:
Okay i didnt see that.

But its still read as part of the `{./utils}` in `_main.cfg` which which should break the schema pass since it adds [event]s at toplevel where they are not supported.

If what i think is true then there are 3 (equally good) ways to hande this:
1) Let the file define a macro with all of these events and use that in these scenarios.
2) Let this file define a  [ressource] with all of these events and use that in these scenarios.
3) Move this file somewhere else where it is not read as part of `{./utils}` (for example a subdiretory of utils)

Reply:
I'll move it to a subdirectory.